### DISCUSSION TITLE
Is there a QR code to make my Go Pro 11 turn on its pairing mode for quik app?

### DISCUSSION THREAD

**Discussion by Telescopictat** [gopro/labs#865] [Q&A]
My rear screen is broken from water damage so it is impossible to use. But I need the rear screen to allow for quick app pairing as I need it to scroll to preferences.

I have installed labs on the go pro and I have tried a QR code from the QRCONTROL app which is named "H8-12 Start Quik App pairing."
When I show the go pro this QR code it makes a small beep and the red light flashes so it recognises that QR code. But on my mobile app it doesnt see my go pro to pair with. 

Is there a better more up to date QR code? Or another workaround to force the Go Pro into quik app pairing mode?

**Comment by dnewman-gpsw** [comment]
Answering here too. Scan this: https://imgur.com/qNnTuoy

**Reply by NunoMig16** [reply]
Hi, can you help me, do you know which is the QR code to pairing the GoPro 9 black with the quick app?

**Reply by dnewman-gpsw** [reply]
If the !PA command linked above didn't work, then HERO9 may not have that support.

**Reply by Jsteiginga** [reply]
> Answering here too. Scan this: https://imgur.com/qNnTuoy

you saved me from a lot of work. Converted my GP to a Naked GP without screen and forgot to turn on pairing, this worked flawlessly, GP11

**Comment by O-Keh-Hunter** [comment]
I almost cried! Thank you so much! Thank you for saving my GoPro 10!

**Reply by ssrossbr** [reply]
Worked with Hero 7 Black

Note: After scan QR Code, press 3 times quickly the shutter button do confirm pairing with Quik App.
